pkgs/applications/networking/mumble/default.nix

```diff
@@ -132,7 +132,10 @@ let
         "-D bundle-qt-translations=OFF"
         "-D bundled-json=OFF"
         "-D bundled-gsl=OFF"
-        # "-D bundled-renamenoise=OFF" # not packaged in nixpkgs yet
+        # upstream  switched back from renamenoise to rnnoise, which is packaged
+        # https://github.com/mumble-voip/mumble/pull/6607
+        # so switch to that as soon as upstream releases
+        # "-D bundled-renamenoise=OFF" # not packaged in nixpkgs
```

Comment:
Why? We don't need that documentation. We don't need to reinvent Mumble documentation. That only adds burden to us maintainers. Remove related commits.

Reply:
[x] done